examples/checker_tests/good/hit_ebpf.stf

```diff
@@ -1,7 +1,7 @@
 # bit<32> A bit<32> B
 # In the output B = (A + 10)
-add pipe_Check_src_ip 0 key.field0:0x0a019845 _NoAction()
-add pipe_Check_src_ip 0 key.field0:0x0a019846 pipe_Reject()
+add pipe.Check_src_ip 0 key.field0:0x0a019845 NoAction()
+add pipe.Check_src_ip 0 key.field0:0x0a019846 pipe.Reject(addr:0)
```

Comment:
Isn't this the same problem that issue870_ebpf.p4 was having?

Reply:
yes; there were a lot of tests like that for some reason. i think p4c has a more flexible grammar for stf than we do, so i change a lot of these underscores to dots and added trivial control plane supplied arguments.